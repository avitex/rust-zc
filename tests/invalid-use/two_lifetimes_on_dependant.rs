use zc::Dependant;

#[derive(Dependant)]
pub struct MyStruct<'a, 'b>(&'a (), &'b ());

fn main() {}
