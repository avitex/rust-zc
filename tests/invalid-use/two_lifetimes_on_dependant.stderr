error: 2 lifetimes on `MyStruct` when only a single is valid on a `zc::Dependant`
 --> $DIR/two_lifetimes_on_dependant.rs:4:20
  |
4 | pub struct MyStruct<'a, 'b>(&'a (), &'b ());
  |                    ^^^^^^^^
